modules/doc_guidelines.adoc

```diff
@@ -0,0 +1,218 @@
+[id="contributing-to-docs-doc-guidelines"]
+= Documentation guidelines
+
+:description: Documentation guidelines for contributing to the Hybrid Cloud Patterns Docs
+:imagesdir: ../images
+
+== General guidelines
+When authoring content, follow these style guides:
+
+* link:https://redhat-documentation.github.io/supplementary-style-guide/[Red Hat Supplementary Style Guide]
+* link:https://www.ibm.com/docs/en/ibm-style[IBM Style Guide], especially link:https://www.ibm.com/docs/en/ibm-style?topic=word-usage[word usage]
++
+NOTE: Use your Red Hat e-mail when asked for IBMid
```

Comment:
Minor suggestion. Also this note wouldn't apply to non-Red Hatters and non-IBMers, right? Maybe we should clarify that.

```suggestion
NOTE: Use your Red Hat e-mail when asked for your IBM ID.
```

Reply:
Reverting to `IBMid` since that's how IBM refers to it